lib/ancestry/class_methods.rb

```diff
@@ -29,27 +29,25 @@ def orphan_strategy= orphan_strategy
 
     # Arrangement
     def arrange options = {}
-      # Get all nodes ordered by ancestry and start sorting them into an empty hash
-      arrange_nodes self.ancestry_base_class.reorder(options.delete(:order)).where(options)
```

Comment:
not sure how removing the ordering helps the pr

The issue with removing this ordering is the tree will still get created, but the nodes will not be in the secondary desired order.

Also of note, the ancestry column does not have `c` collation. So I do not trust sql's `order` right now.